### PR TITLE
Refactor/chain/detele

### PR DESCRIPTION
**Pull request by ramirobg94**

### CODE REVIEW

File: webapp/store/survey/actions/actionTypes.js [hunk 1]
```diff
@@ -3,3 +3,5 @@ export const surveyUpdate = 'survey/update'
 export const surveyDelete = 'survey/delete'
 export const surveyDefsLoad = 'survey/defs/load'
 export const surveyDefsReset = 'survey/defs/reset'
+export const surveyChainSave = 'survey/chain/save'
+export const surveyChainElementDelete = 'survey/chain/delete'
```
Comment (minotogna) [inline webapp/store/survey/actions/actionTypes.js:7]: this is better: ```export const surveyChainItemDelete = 'survey/chain/item/delete'```

File: webapp/store/survey/actions/chainElementDelete.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import * as ActionTypes from './actionTypes'
+
+export const chainElementDelete = () => ({ type: ActionTypes.surveyChainElementDelete })
```
Comment (minotogna) [inline webapp/store/survey/actions/chainElementDelete.js:3]: ```export const chainItemDelete = () => ({ type: ActionTypes.surveyChainElementDelete })```

and add it to webapp/store/survey/actions/chain.js

File: webapp/store/survey/actions/chainSave.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import * as ActionTypes from './actionTypes'
+
+export const chainSave = () => ({ type: ActionTypes.surveyChainSave })
```
Comment (minotogna) [inline webapp/store/survey/actions/chainSave.js:3]: add this to webapp/store/survey/actions/chain.js

File: webapp/store/survey/actions/actionTypes.js (hunk 1 above)
Comment (ramirobg94) [reply webapp/store/survey/actions/actionTypes.js:7]: Thanks, I had some bad feelings when I put that name

**Pull request merged**